tests/integration/auth.spec.coffee

```diff
@@ -109,7 +129,42 @@ describe 'SDK authentication', ->
 
 		givenLoggedInUser()
 
-		describe 'resin.auth.isLogged()', ->
+		describe 'resin.auth.isLoggedIn()', ->
+
+			it 'should eventually be true', ->
+				promise = resin.auth.isLoggedIn()
+				m.chai.expect(promise).to.eventually.be.true
+
+		describe 'resin.auth.logout()', ->
+
+			it 'should logout the user', ->
+				resin.auth.logout().then ->
+					promise = resin.auth.isLoggedIn()
+					m.chai.expect(promise).to.eventually.be.false
```

Comment:
NIT: I would love to also check the result of `getToken` in here as well, since this isn't tested anywhere atm.

Reply:
@thgreasi it's tested on line 32

Reply:
@thgreasi , or did you mean that I should check it after logging out?

Reply:
What I meant was to check that `getToken` behaves as expected right after a `logout`.

In [line 32](https://github.com/resin-io/resin-sdk/blob/21674ac53527c7f7dacac6510f4b6895af11a11c/tests/integration/auth.spec.coffee#L32-L47)  `getToken` is called after a `loginWithToken()`. The case I was talking about is the supplementary one, to ensure that no token is returned after a user logs out.